data_structures/linked_list/doublyLinkedList.c

```diff
@@ -0,0 +1,96 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+struct node {
+    int num;
+    struct node * preptr;
+    struct node * nextptr;
+}*stnode, *ennode;
+ 
+
+void DlListcreation(int n);
```

Comment:
Could you replace DlListcreation by `void addNode (int n)`, and `int removeNode(node *&head)`?